tests/testsuite/help.rs

```diff
@@ -17,9 +15,15 @@ fn help() {
     cargo_process("build -h").run();
     cargo_process("help help").run();
     // Ensure that help output goes to stdout, not stderr.
-    cargo_process("search --help").with_stderr("").run();
+    cargo_process("search --help").with_stderr_data("").run();
     cargo_process("search --help")
-        .with_stdout_contains("[..] --frozen [..]")
+        .with_stdout_data(
```

Comment:
We can probably delete this test because we have help UI tests for every command, like

https://github.com/rust-lang/cargo/blob/master/tests/testsuite/cargo_search/help/mod.rs

Reply:
Thanks. Removed in eef057e.